upgrade/Upgrade-3.4.6.php

```diff
@@ -0,0 +1,16 @@
+<?php
+if (!defined('_PS_VERSION_'))
+    exit;
+
+function upgrade_module_3_4_6($object)
```

Comment:
I guess your function name is wrong, because I see that the new version number is 3.6.4 in your main class. ;)